packages/sku/lib/runESLint.js

```diff
@@ -59,6 +59,8 @@ const runESLint = async ({ fix = false, paths }) => {
       if (e && e.message && e.message.includes('No files matching')) {
         console.warn(yellow(`Warning: ${e.message}`));
       } else {
+        console.warn(yellow('ESLint encountered an error:'));
+        console.log(e.message);
```

Comment:
Do we want to `console.error` here instead of just `warn`/`log`?

Reply:
Happy for it to be either, but `console.log` isn't as noisy (red).